### ISSUE TITLE
Query : Use of _.debounce 

### ISSUE THREAD

**Issue by nandita121189** [readium/readium-shared-js#397]
Can you please explain the use of the following line in `reflowable_view.js`

```
_.debounce(updatePagination_, 100);
```
This line is causing the `updatePagination_` function to be called twice on window resize, due to which a flickering is observed after the window resizes. 

If I rewrite the code after removing the `_.debounce`, then flickering no longer occurs.
```
 var updatePagination = updatePagination_; 
```

Can you please guide me if rewriting the code can cause any regression issues. ?

**Comment by danielweck** [comment]
This use of the `_.debounce()` function ensures that the `updatePagination_()` function is executed *only once* even though client code may invoke the debounced `updatePagination()` function several times in quick successions, before the 100ms time window elapses.
The "flickering" you are referring to seems to be the result of a 100ms period of "inactivity", as the `updatePagination_()` function is effectively not called instantly.

If you remove the "debouncer" you may indeed observe the immediate benefit of instant refresh, but in some other (edge?) cases the multiple quick-fire calls to `updatePagination()` will create worse problems.

So, could you please try to add the "true" function parameter after "100", so that the `updatePagination_()` function is invoked on the leading edge of the wait interval (immediately), rather than the trailing edge (deferred).
http://underscorejs.org/#debounce

Many thanks! :)

**Comment by nandita121189** [comment]
Thanks @danielweck , the solution you suggested worked. Now the `updatePagination_  `function is called only once on window resize and flickering doesn't occur.

**Comment by rkwright** [comment]
Appears resolved.  Closing

**Comment by danielweck** [comment]
Issue "migrated" to: https://github.com/readium/readium-shared-js/issues/399
